source/guides/clear/mixer.rst

```diff
@@ -913,154 +956,74 @@ precedence over any upstream bundles that have the same name. This
 precedence enables you to copy upstream bundles locally, and edit into a
 local variation.
 
-Bundle configuration
---------------------
-
-mixer provides commands to configure the bundles for a mix, such as to add a
-bundle to a mix, to create a new bundle for a mix, or to remove a bundle from a
-mix. View the `mixer.bundle man page`_ for a full list of commands and more
-information on configuring bundles in a mix.
-
-Editing an existing local bundle is as simple as opening the bundle definition
-file in your favorite editor, making the desired edits, and saving your changes.
-
-.. note::
-
-   Removing bundles from a mix: By default, removing a bundle will only
-   remove the bundle from the mix. The local bundle definition file will
-   still remain. To completely remove a bundle, including its local bundle definition file, use the :command:`--local` flag.
 
-   If you remove the bundle definition file for a local, edited version of an
-   upstream bundle in a mix, the mix reverts to reference the original upstream version of the bundle.
+Bundle definition files
+-----------------------
 
-.. rst-class:: content-collapse
+A ``bundle definition`` file consists of a header, followed by a list 
+of packages and directives. The header holds important meta-data, like 
+the TITLE, DESCRIPTION, and STATUS. Other meta-data include TAGS, which 
+define a bundle's function in the ecosystem, and MAINTAINER, which gives 
+contact information. 
 
-Configure and enable Docker
-===========================
+Following the header are the directives, shown in Table 2. 
 
-Use these steps to enable Docker for the mixer tool. Make sure to
-`Configure Docker proxy info`_ first if needed.
+.. list-table:: **Table 2**: Bundle directives
+   :widths: 50,50
+   :header-rows: 1 
 
-#. Start the Docker daemon:
+   * - **Directive**
+     - **Explanation**
```

Comment:
```suggestion
     - **Description**
```

Reply:
I'm OK with changing this Column header to "Description". However, if we change this, then the Column header in Table 1, "Explanation", should also be changed.  I changed the latter.